spec/rubocop/cop/layout/empty_line_after_guard_clause_spec.rb

```diff
@@ -199,6 +199,28 @@ def foo
     RUBY
   end
 
+  it 'registers and corrects when using guard clause is after `rubocop:disable` comment' do
```

Comment:
Theres should also be a test making sure an empty line is required after the enable directive.